### DISCUSSION TITLE
自建服务器，开始连接时经常要等约半分钟才能连接上

### DISCUSSION THREAD

**Discussion by Green1ee** [rustdesk/rustdesk#860] [Q&A]
自建服务器，开始连接时经常要等约半分钟才能连接上，一但连接上使用又比较顺畅，不知道是为何

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/815

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/861
